index.html

```diff
@@ -30,16 +44,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id="teal" class="box">Box 1</div>
+                <div id="gold" class="box">Box 2</div>
+                <div id="cadetblue" class="box">Box 3</div>
+                <div id="coral" class="box">Box 4</div>
+                <div id="crimson" class="box">Box 5</div>
+                <div id="forestgreen" class="box">Box 6</div>
+                <div id="darkorchid" class="box">Box 7</div>
+                <div id="hotpink" class="box">Box 8</div>
+                <div id="indigo" class="box">Box 9</div>
+                <div id="dodgerblue" class="box">Box 10</div>
```

Comment:
Try to not use ID tags so much although in this case would be fine, but using class would give you the same effect. For example https://codepen.io/casta/pen/VqVRzB I posted a codepen to explain a little bit more of what I'm talking. If no other option, use ID as it is more specific, but try and use class as much as possible.